console/src/components/Services/Data/TableBrowseRows/FilterActions.js

```diff
@@ -108,7 +108,7 @@ const runQuery = tableSchema => {
       delete newQuery.order_by;
     }
     dispatch({ type: 'ViewTable/V_SET_QUERY_OPTS', queryStuff: newQuery });
-    dispatch(vMakeRequest());
+    dispatch(vMakeRequest()).then(() => dispatch(vMakeCountRequest()));
```

Comment:
This is done twice, here and once in `ViewTable` as well.
Do you think there should be a function called `vMakeTableRequests` that abstracts out this part? Something like:

```js
const vMakeTableRequests = () => (dispatch, getState) => {
  dispatch(vMakeRequest()).then(() => dispatch(vMakeCountRequest()));
};
```
You can then call this one function everywhere. Later, if somebody is iterating some part of this code, they might not know to make the change in both places.